### PR TITLE
Close #7784: i18n: The alt text for image is translated by default

### LINKED ISSUE #7784: No translation for :alt: of images
**Describe the bug**
Text of :alt: parameter of an image is not considered.

**To Reproduce**

Create a project, add an image directive with :alt: text and do `make gettext`.

**Expected behavior**

The text of :alt: property will be in the .pot file.

**Environment info**
- OS: Linux OpenSuSE 15.2
- Python version: 3.6
- Sphinx version: 3.x and master branches
- Sphinx extensions:  No extension
- Extra tools: None

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Feature

### Purpose
- refs: #7784 
- Make alt text for image translatable by default without settings of
gettext_additional_targets.

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -7,6 +7,8 @@ Dependencies
 Incompatible changes
 --------------------
 
+* #7784: i18n: The msgid for alt text of image is changed
```
Comment (tk0miya) [inline CHANGES:15]: This change brings a breaking change. So it might be better to merge this into 4.0.
@shimizukawa any comments?

Comment (shimizukawa) [reply CHANGES:15]: Good.

**Review by shimizukawa — APPROVED** [review]
LGTM!

File: CHANGES (hunk 1 above)
Comment (tk0miya) [reply CHANGES:15]: Okay, rebasing now.

**Pull request merged**